lib/elixir/lib/string.ex

```diff
@@ -599,11 +599,15 @@ defmodule String do
       "   abc "
 
   """
-  @spec rstrip(t, char) :: t
+  @spec rstrip(t, char | binary) :: t
   def rstrip(string, char) when is_integer(char) do
     replace_trailing(string, <<char::utf8>>, "")
   end
 
+  def rstrip(string, to_strip) when is_binary(to_strip) do
+    String.replace_trailing(string, to_strip, "")
```

Comment:
You can omit the `String.` here since the `replace_trailing/3` function is in the same module.